contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -134,21 +132,20 @@ contract VaultAdmin is VaultStorage {
     }
 
     /**
-     * @dev Set maximum amount of OUSD that can at any point be minted and deployed
+     * @dev Set maximum amount of OTokens that can at any point be minted and deployed
      * to strategy (used only by ConvexOUSDMetaStrategy for now).
-     * @param _threshold OUSD amount with 18 fixed decimals.
+     * @param _threshold OToken amount with 18 fixed decimals.
      */
-    function setNetOusdMintForStrategyThreshold(uint256 _threshold)
-        external
-        onlyGovernor
-    {
+    function setNetOusdMintForStrategyThreshold(
```

Comment:
At some point we will need to change the name of this function too. And deploy an updated version of the strategies it will affect.

Reply:
Since this is only called by the governor, I think we can change it.
I'll come up with a sensible name tomorrow

Reply:
Ok, I've created a new PR that will change the function names and events with `Ousd` in them https://github.com/OriginProtocol/origin-dollar/pull/1629
The new PR merges to the branch this PR is using.

I'll cover renaming the storage variables in a separate PR as that is riskier.

Reply:
PR https://github.com/OriginProtocol/origin-dollar/pull/1629 will not be merged before the OETH Vault redeployment for collateral swaps.